scripts/superdesk-ingest/module.js

```diff
@@ -1435,17 +1435,15 @@
                 icon: 'archive',
                 monitor: true,
                 controller: ['api', 'data', 'desks', function(api, data, desks) {
-                    desks.fetchCurrentDeskId().then(function(deskid) {
-                        api(data.item.fetch_endpoint).save({
+                    return desks.fetchCurrentDeskId().then(function(deskid) {
+                        return api(data.item.fetch_endpoint).save({
                             guid: data.item.guid,
                             desk: deskid
                         })
-                        .then(
-                            function(response) {
-                                data.item.error = response;
-                            })
-                        ['finally'](function() {
-                            data.item.actioning.externalsource = false;
+                        .then(function(response) {
+                            data.item.error = response;
```

Comment:
this seems like it should go into error handler, and not into success response handler